bigquery/google/cloud/bigquery/job.py

```diff
@@ -1541,9 +1541,11 @@ def destination_uri_file_counts(self):
         See:
         https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs#statistics.extract.destinationUriFileCounts
 
-        :rtype: int or None
-        :returns: number of DML rows affectd by the job, or None if job is not
-                  yet complete.
+        :rtype: list of int or None
```

Comment:
Could we change this to a [Napoleon/Google-style](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html#example-google) docstring while we're in here?